src/poetry/core/version/pep440/segments.py

```diff
@@ -7,23 +7,23 @@
 from typing import Union
 
 
-RELEASE_PHASE_ALPHA = "alpha"
-RELEASE_PHASE_BETA = "beta"
-RELEASE_PHASE_RC = "rc"
-RELEASE_PHASE_PREVIEW = "preview"
-RELEASE_PHASE_POST = "post"
-RELEASE_PHASE_REV = "rev"
-RELEASE_PHASE_DEV = "dev"
-RELEASE_PHASES = {
-    RELEASE_PHASE_ALPHA: "a",
-    RELEASE_PHASE_BETA: "b",
-    RELEASE_PHASE_RC: "c",
-    RELEASE_PHASE_PREVIEW: "pre",
-    RELEASE_PHASE_POST: "-",  # shorthand of 1.2.3-post1 is 1.2.3-1
-    RELEASE_PHASE_REV: "r",
-    RELEASE_PHASE_DEV: "dev",
+# Release phase IDs according to PEP440
+RELEASE_PHASE_ID_ALPHA = "a"
+RELEASE_PHASE_ID_BETA = "b"
+RELEASE_PHASE_ID_RC = "rc"
+RELEASE_PHASE_ID_POST = "post"
+RELEASE_PHASE_ID_DEV = "dev"
+
+RELEASE_PHASE_SPELLINGS = {
+    RELEASE_PHASE_ID_ALPHA: {"a", "alpha"},
```

Comment:
Might be good to reuse the vars here for "a" here and similar for the rest. Nitpick.